.github/workflows/import-l10n.yml

```diff
@@ -9,8 +9,6 @@ permissions:
   pull-requests: write
 
 on:
-  schedule:
-    - cron: "0 0 * * *"
```

Comment:
We should keep this enabled, otherwise the ac-sync-strings.yml won't have anything to uplift.

Reply:
Thanks for pointing that out!  Updated.

Reply:
There is also a parallel conversation on Slack.

If we want to keep main frozen, we could:
* Disable this workflow (as you did)
* Update the import workflow to write to the release 125 branch. It needs to be tested, but updating the branch reference [here](https://github.com/mozilla-mobile/firefox-android/blob/main/.github/workflows/import-l10n.yml#L25) might be all we need.